mapit_gb/data/authorities.json

```diff
@@ -0,0 +1,2217 @@
+    "name": "Dumfries and Galloway Council",
+    "ons": "00QH",
+    "gss": "S12000006"
+  },
+  "portsmouth": {
+    "name": "Portsmouth City Council",
+    "ons": "00MR",
+    "gss": "E06000044"
+  },
+  "plymouth": {
+    "name": "Plymouth City Council",
+    "ons": "00HG",
+    "gss": "E06000026"
+  },
+  "isle-of-wight": {
+    "name": "Isle of Wight Council",
+    "ons": "00MW",
+    "gss": "E06000046"
+  },
+  "isle-of-anglesey": {
+    "name": "Isle of Anglesey Council",
+    "ons": "00NA",
+    "gss": "W06000001"
+  },
+  "gwynedd": {
+    "name": "Gwynedd Council",
+    "ons": "00NC",
+    "gss": "W06000002"
+  },
+  "derry": {
+    "name": "Derry City Council",
+    "ons": "95A",
+    "gss": null
+  },
+  "limavady": {
+    "name": "Limavady Borough Council",
+    "ons": "95B",
+    "gss": null
+  },
+  "coleraine": {
+    "name": "Coleraine Borough Council",
+    "ons": "95C",
+    "gss": null
+  },
+  "ballymoney": {
+    "name": "Ballymoney Borough Council",
+    "ons": "95D",
+    "gss": null
+  },
+  "moyle": {
+    "name": "Moyle District Council",
+    "ons": "95E",
+    "gss": null
+  },
+  "larne": {
+    "name": "Larne Borough Council",
+    "ons": "95F",
+    "gss": null
+  },
+  "ballymena": {
+    "name": "Ballymena Borough Council",
+    "ons": "95G",
+    "gss": null
+  },
+  "magherafelt": {
+    "name": "Magherafelt District Council",
+    "ons": "95H",
+    "gss": null
+  },
+  "cookstown": {
+    "name": "Cookstown District Council",
+    "ons": "95I",
+    "gss": null
+  },
+  "strabane": {
+    "name": "Strabane District Council",
+    "ons": "95J",
+    "gss": null
+  },
+  "omagh": {
+    "name": "Omagh District Council",
+    "ons": "95K",
+    "gss": null
+  },
+  "fermanagh": {
+    "name": "Fermanagh District Council",
+    "ons": "95L",
+    "gss": null
+  },
+  "dungannon-and-south-tyrone": {
+    "name": "Dungannon and South Tyrone Borough Council",
+    "ons": "95M",
+    "gss": null
+  },
+  "craigavon": {
+    "name": "Craigavon Borough Council",
+    "ons": "95N",
+    "gss": null
+  },
+  "armagh": {
+    "name": "Armagh District Council",
+    "ons": "95O",
+    "gss": null
+  },
+  "newry-and-mourne": {
+    "name": "Newry and Mourne District Council",
+    "ons": "95P",
+    "gss": null
+  },
+  "banbridge": {
+    "name": "Banbridge District Council",
+    "ons": "95Q",
+    "gss": null
+  },
+  "down": {
+    "name": "Down District Council",
+    "ons": "95R",
+    "gss": null
+  },
+  "lisburn": {
+    "name": "Lisburn Borough Council",
+    "ons": "95S",
+    "gss": null
+  },
+  "antrim": {
+    "name": "Antrim Borough Council",
+    "ons": "95T",
+    "gss": null
+  },
+  "newtownabbey": {
+    "name": "Newtownabbey Borough Council",
+    "ons": "95U",
+    "gss": null
+  },
+  "carrickfergus": {
+    "name": "Carrickfergus Borough Council",
+    "ons": "95V",
+    "gss": null
+  },
+  "north-down": {
+    "name": "North Down Borough Council",
+    "ons": "95W",
+    "gss": null
```

Comment:
We're not importing these slugs because they don't have GSS codes, so I wonder if we might as well delete them from the file?

Pro: smaller file, less to get confused about if you're not also looking at the importer
Con: we lose the history and the old slugs which would have existed as pages somewhere - perhaps if we do delete them we do it in a separate commit that only does the deletion so that there's some history.